api/DOMRect.json

```
@@ -225,6 +225,104 @@
           }
         }
       },
+      "height": {
+        "__compat": {
+          "spec_url": "https://drafts.fxtf.org/geometry/#dom-domrectreadonly-height",
+          "support": {
+            "chrome": {
+              "version_added": "61"
+            },
+            "chrome_android": {
+              "version_added": "61"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": "27"
+            },
+            "firefox_android": {
+              "version_added": "27"
+            },
+            "ie": {
+              "version_added": false,
```

```suggestion
              "version_added": "4",
```

And no note needed. These were copied from `DOMRectReadOnly.json` but I think we should remove everything about `ClientRect` from there, including the notes.

---

I've sent https://github.com/mdn/browser-compat-data/pull/11694 for that.